test/e2e/multi-cluster-deployment/run-test.sh

```diff
@@ -48,27 +48,21 @@ while [ $# != 0 ]; do
     shift
 done
 
-set -e # exit on error
+case "$env" in
+    (kind|ocp) ;;
+    (*) echo "$0: --env must be 'kind' or 'ocp'" >&2
+        exit 1;;
+esac
 
-if [ $env == "kind" ];then
-    SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
-    COMMON_SRCS="${SRC_DIR}/../common"
-    HACK_DIR="${SRC_DIR}/../../../hack"
+set -e # exit on error
 
-    "${HACK_DIR}/check_pre_req.sh" --assert --verbose kubectl docker kind make go ko yq helm kflex ocm
+SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
+COMMON_SRCS="${SRC_DIR}/../common"
+HACK_DIR="${SRC_DIR}/../../../hack"
 
-    "${COMMON_SRCS}/cleanup.sh"
-    source "${COMMON_SRCS}/setup-shell.sh"
-    "${COMMON_SRCS}/setup-kubestellar.sh" $setup_flags
-    "${SRC_DIR}/use-kubestellar.sh"
+"${HACK_DIR}/check_pre_req.sh" --assert --verbose kubectl docker kind make go ko yq helm kflex ocm
```

Comment:
Don't we run this also for the "release" tests?  In that case I don't think we require the ability to build, so I'm not sure all these packages are mandatory. Enforcing it here may contradict our docs for users.

Reply:
This list of checked prereqs is not being changed in this PR; can we address it in a follow-on PR?